src/gui/guiFormSpecMenu.h

```diff
@@ -32,6 +32,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/string.h"
 #include "util/enriched_string.h"
 #include "StyleSpec.h"
+#include "client/texture_pool.h"
```

Comment:
Move the include into the .cpp file and add a `class TexturePool;` to the header (below this comment).